api/src/main/java/com/epam/pipeline/controller/cluster/ClusterController.java

```diff
@@ -286,4 +288,41 @@ public Result<InstanceDNSRecord> requestDnsRecord(@RequestParam final Long regio
                                                       @RequestBody final InstanceDNSRecord dnsRecord) {
         return Result.success(infrastructureApiService.createInstanceDNSRecord(regionId, dnsRecord));
     }
+
+    @GetMapping("/cluster/core/pods")
```

Comment:
I think is it better to swap `core` and `pods` to get the same order as in `/cluster/pods/info`